contracts/RexCaptain.sol

```diff
@@ -425,6 +418,7 @@ contract REXCaptain is AccessControlEnumerable, SuperAppBase {
         voteInProgress(captainAddress)
     {
         RexCaptainStorage.Vote storage currentVote = _getCurrentVote(captainAddress);
+        // TODO: Require vote exsits?
```

Comment:
`L422` is already checking for vote exists.

```
require(currentVote.voted[msg.sender] == false, "Already voted");
```

Reply:
:+1: